dataminer-overview/Support Services/User_Operations.Support.md

```diff
@@ -0,0 +1,81 @@
+---
+uid: User_operations_support
+---
+
+# User Operations Support application
+
+With the User Operations Support application, you can both keep track of existing tickets and create new ones.
+
+The application is available at <https://operations.skyline.be/>
```

Comment:
Can anyone log in to this application? Or does logging on happen [in the same way as for dataminer.services](https://docs.dataminer.services/user-guide/Cloud_Platform/AboutCloudPlatform/Logging_on_to_the_DataMiner_Cloud_Platform.html#logging-on)?

Reply:
Same as dataminer.services